### DISCUSSION TITLE
Telegram

### DISCUSSION THREAD

**Discussion by Wierzciu** [chriskyfung/AutoFetcher-IG-Stories-to-GDrive#123] [Ideas]
Hello,
do You plan to release Telegram version?

**Comment by chriskyfung** [comment]
Hi @Wierzciu, 

Thanks for reaching out! I don't have such vision right now, mainly because we're not regular Telegram users. Therefore, I'm not sure what features would be needed. I appreciate the suggestion and might consider it if more people show interest.
